src/Exceptionless.Core/Repositories/Configuration/Indexes/OrganizationIndex/OrganizationIndex.cs

```diff
@@ -19,8 +21,8 @@ public sealed class OrganizationIndex : VersionedIndex {
         public override CreateIndexDescriptor ConfigureIndex(CreateIndexDescriptor idx) {
             return base.ConfigureIndex(idx.Settings(s => s
                 .Analysis(d => d.Analyzers(b => b.Custom(KEYWORD_LOWERCASE_ANALYZER, c => c.Filters("lowercase").Tokenizer("keyword"))))
-                .NumberOfShards(_settings.ElasticsearchNumberOfShards)
-                .NumberOfReplicas(_settings.ElasticsearchNumberOfReplicas)
+                .NumberOfShards(_elasticsearchOptions.Value.NumberOfShards)
```

Comment:
Maybe be less verbose all over the place and just call it _options and know that it is options for the class that we are in. Don't really like us having to access both AppOptions and ElasticsearchOptions. We should just add ScopePrefix to ElasticsearchOptions.

Reply:
but we have app scope and that affects other things

Reply:
Ok, what's your point? An option can be in more than one options class.

Reply:
If we do that it seems like we should add scope to storage as well

Reply:
That’s fine. Again, we are trying to simplify. I don’t like having multiple options classes in a single class and I want to simplify things as much as possible.